app/src/main/java/io/appium/uiautomator2/model/api/touch/PackageModel.java

```diff
@@ -0,0 +1,31 @@
+package io.appium.uiautomator2.model.api.touch;
+
+public class PackageModel {
+    String PackageName;
```

Comment:
member names should be in `camelCase`

Reply:
you could also make them `private final` as they are not supposed to be mutable during model lifecycle

Reply:
this is still relevant. I don't see a reason to make model fields mutable

Reply:
Done please verify